specification/apimanagement/control-plane/Microsoft.ApiManagement/preview/2017-03-01/apimidentityprovider.json

```diff
@@ -0,0 +1,455 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ApiManagementClient",
+    "description": "Use these REST APIs for performing operations on Identity Provider entity associated with your Azure API Management deployment. Setting up an external Identity Provider for authentication can help you manage the developer portal logins using the OAuth2 flow.",
+    "version": "2017-03-01"
+  },
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{apimBaseUrl}",
+    "useSchemePrefix": false,
+    "positionInOperation": "first",
+    "parameters": [
+      {
+        "name": "apimBaseUrl",
+        "description": "The management endpoint of the API Management service, for example https://myapimservice.management.azure-api.net.",
+        "required": true,
+        "type": "string",
+        "in": "path",
+        "x-ms-skip-url-encoding": true
+      }
+    ]
+  },
+  "schemes": [
+    "https"
+  ],
+  "securityDefinitions": {
+    "apim_key": {
+      "type": "apiKey",
+      "name": "Authorization",
+      "in": "header"
+    }
+  },
+  "security": [
+    {
+      "apim_key": []
+    }
+  ],
+  "paths": {
+    "/identityProviders": {
+      "get": {
+        "tags": [
+          "IdentityProvider"
+        ],
+        "operationId": "IdentityProvider_List",
```

Comment:
>IdentityProvider_List [](start = 24, length = 21)

Pageable?

Reply:
not required, their are only 4 identity providers and would never be more than 10

---
In reply to: [163359070](https://github.com/Azure/azure-rest-api-specs/pull/2269#discussion_r163359070) [](ancestors = 163359070)

Reply:
An operation can be pageable even if it does not really have a `nextPage`. Take a look at [example 3](https://github.com/Azure/autorest/tree/master/docs/extensions#x-ms-pageable). The extension still can be useful at code generation time.

Reply:
fixed